doc/whats_new/v1.3.rst

```diff
@@ -136,6 +143,12 @@ Changelog
   scikit-learn 1.3: retraining with scikit-learn 1.3 is required.
   :pr:`25186` by :user:`Felipe Breve Siola <fsiola>`.
 
+- |Fix| A deprecation warning is raised when using the "base_estimator__" prefix
```

Comment:
Then, I think that we don't need this entry. We can alterate the previous 1.2 entry mentioning that the warning is also announce when calling `set_params`.

Reply:
I think this need to me moved to a dedicated entry targeting the 1.2.2 version.

Reply:
I've moved the change log entries to the 1.2.2 what's new. So we now have a dedicated entry.